.github/workflows/test.yaml

```diff
@@ -16,8 +16,8 @@ jobs:
       uses: actions/setup-go@v2-beta
     - name: Install modules
       run: make install-tools
-    - name: Check License Headers
-      run: make license-header-check
+    #- name: Check License Headers
+    #  run: make license-header-check
```

Comment:
Why is the license header check removed in this PR?

Reply:
A left-pad esque issue with add license (doesn't even work with checked out versions that have worked on past commits).
https://github.com/nitrictech/go-sdk/runs/3286678696?check_suite_focus=true

Reply:
sorted it was an issue with a later go version, have cemented it in CI for now.